dev-docs/add-a-prebid-server-adapter.md

```diff
@@ -2,7 +2,7 @@
 layout: page
 title: Add a Bidder Adapter to Prebid Server
 description: Add a Bidder Adapter to Prebid Server
-pid: 28
+pid: 29
```

Comment:
what's this do? @jeanstemp asked me about it and I wasn't sure.

Reply:
As far as I can tell, it's just used to sort pages in the navbar. I thought:

```
getting started with prebid server
prebid server bidders
add a new prebid server bidder
```

Made a bit more sense than

```
getting started with prebid server
add a new prebid server bidder
prebid server bidders
```

Reply:
makes sense, thanks